include/lsst/meas/base/ShapeUtilities.h

```diff
@@ -130,7 +130,8 @@ class ShapeResultKey : public afw::table::FunctorKey<ShapeResult>  {
         afw::table::Schema & schema,
         std::string const & name,
         std::string const & doc,
-        UncertaintyEnum uncertainty
+        UncertaintyEnum uncertainty,
+        afw::table::CoordinateType coordType=afw::table::CoordinateType::PIXEL
```

Comment:
In C++11, it's permitted to give scope to a regular `enum`. That is, `afw::table::CoordinateType::PIXEL` and `afw::table::PIXEL` are equivalent, and this does, in fact, work. I agree that this code is ugly & inconsistent, though.

It looks as though Swig is happy with `enum class`, so I think switching everything to that is a clear win.